docs/examples/api-reference/client/branch.py

```diff
@@ -55,4 +55,12 @@ def unused_checkout():
 
         # all subsequent operations will be on `mybranch`
         client.commit(...)
+
+        # create and change active branch from `mybranch` to `mybranch2`
+        # docsnip-highlight start
+        client.checkout("mybranch2", True)
```

Comment:
In code snippets, we should pass this init via kwarg instead of args. Basically whatever examples we write, people copy that and so we need our examples to set best practices on readability. 

`client.checkout("mybranch2", init=True)`